context_test.go

```diff
@@ -0,0 +1,78 @@
+package test
+
+import (
+	"context"
+	"testing"
+)
+
+type key int
+
+var ctxKey key = 1
+
+func ContextKey(ctx context.Context) (string, bool) {
+	if v := ctx.Value(ctxKey); v != nil {
+		return v.(string), true
+	}
+	return "", false
+}
```

Comment:
this function is a test fixture, currently only used with test values known to be of type string.  This will be addressed if necessary in the future if it becomes necessary to avoid such panics as the result of additional test cases.